template.yaml

```diff
@@ -234,6 +241,7 @@ Resources:
               - sts:AssumeRole
       ManagedPolicyArns:
         - arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+      PermissionsBoundary: !If [CreateDefaultBoundaryPolicy, !Ref AWS::NoValue, !Ref PermissionBoundaryPolicyArn]
```

Comment:
Is it needed to have this here as well as in the globals section? If it's superfluous, best remove it.